src/lang/en-US.json

```diff
@@ -184,6 +184,12 @@
   "filter_last_month": "Last month",
   "filter_last_3_months": "Last 3 months",
   "filter_last_6_months": "Last 6 months",
+  "filter_with_games_played": "Games played",
```

Comment:
Possibly change this to "At least this many games played" and then each item in the list doesn't need its own string (you can just use the number 5/10/15...)